FusionIIIT/applications/visitor_hostel/migrations/0002_auto_20230401_1704.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.1.5 on 2023-04-01 16:04
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('visitor_hostel', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='bill',
+            name='bill_date',
+            field=models.DateField(blank=True, default=django.utils.timezone.now, blank = True),
+        ),
+        migrations.DeleteModel(
```

Comment:
@VedantJain1103 Please do the same as you have done for class Bill, you are again deleting Meal Record and recreating it.